LayoutTests/fast/preloader/image-with-incorrect-tag.html

```diff
@@ -0,0 +1,11 @@
+<body>
+    <script>
+    if (window.testRunner) {
+        testRunner.dumpAsText();
+        testRunner.dumpResourceResponseMIMETypes();
+    }
+    </script>
+    This test requires DumpRenderTree to see the log of what resources are loaded.
+    <script src=resources/non-existant.js></script>
```

Comment:
@Ahmad-S792 Can you kindly clarify the expectation of this line? dumpResourceResponseMIMETypes does not filter out missing resources from output, so, "non-existant.js has MIME type application/x-javascript" would be expected to be shown as there is not any tag like template wrapping this element that would prevent this resource to load.